gson/src/main/java/com/google/gson/FieldNamingPolicy.java

```diff
@@ -75,6 +75,24 @@
     }
   },
 
+    /**
```

Comment:
Indentation seems wrong here.

Reply:
Ah! Good catch, don't know how I missed it...it's fixed now. And thank you for reviewing it so quickly!